locations/spiders/safeway.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import re
+
+from locations.items import GeojsonPointItem
+
+
+class SafewaySpider(scrapy.Spider):
+    name = "safeway"
+    allowed_domains = ['safeway.com']
+    start_urls = (
+        'https://local.safeway.com/sitemap.xml',
+    )
+
+    def parse(self, response):
+        response.selector.remove_namespaces()
+        city_urls = response.xpath('//*/*[@href]').extract()
+        for path in city_urls:
+            locationURL = re.compile(r'https://local.safeway.com/(safeway/|\S+)/\S+/\S+/\S+.html')
+            if not re.search(locationURL, path):
+                pass
+            else:
+                path = re.search(locationURL, path)[0].strip('"/>')
+                yield scrapy.Request(
+                    path.strip(),
+                    callback=self.parse_store,
+                )
+
+    def parse_store(self, response):
+
+        properties = {
+            'name': response.xpath('/html/body/main/div[2]/div/section[1]/div[2]/div[1]/h1/span/span[2]/text()').extract_first(),
+            'website': response.request.url,
+            'ref': response.xpath('/html/body/main/div[2]/div/section[1]/div[2]/div[1]/h1/span/span[2]/text()').extract_first(),
+            'opening_hours': " ".join(response.xpath('/html/body/main/div[2]/div/section[1]/div[2]/div[2]/div[1]/div[2]/div[@data-days]').extract_first().split("data-days=\'")[1].split("\'")[0].replace(']','').replace('{','').replace('"day"','').replace('"intervals"','').replace(':','').replace('"',' ').replace(',','').replace('[','').replace('}','').replace('end','close').replace('start','open').split()),
```

Comment:
Please correct the opening hours format. See https://github.com/alltheplaces/alltheplaces/pull/254#issuecomment-349794850 for details on the format.